src/main/scala/viper/silver/ast/Statement.scala

```diff
@@ -197,6 +197,34 @@ case class Goto(target: String)(val pos: Position = NoPosition, val info: Info =
   */
 case class LocalVarDeclStmt(decl: LocalVarDecl)(val pos: Position = NoPosition, val info: Info = NoInfo, val errT: ErrorTrafo = NoTrafos) extends Stmt
 
+/** Havoc statement.
+  * Replaces the havocked resources snapshot with a fresh snapshot.
+  * The optional lhs provides a guard, under which the resource is havocked. For example,
+  *   havoc c ==> Pred(x)
+  * means that we only havoc Pred(x) if the condition `c` is true.
+  * The "havocall" variant of this statement allows one to havoc an unbounded number of resources
+  */
+case class Havoc(lhs: Option[Exp], exp: ResourceAccess)(val pos: Position = NoPosition, val info: Info = NoInfo, val errT: ErrorTrafo = NoTrafos) extends Stmt {
+  override lazy val check : Seq[ConsistencyError] = {
```

Comment:
Surplus space in `check : Seq` (here and further down). Format analogous to other types, as `check: Seq`